### DISCUSSION TITLE
Attempt AccessTokens auth login

### DISCUSSION THREAD

**Discussion by SzAkos** [codeigniter4/shield#705] [Q&A] [answered]
Hey,

I'm using this snippet to login from application:
https://codeigniter4.github.io/shield/guides/mobile_apps/

When I try to login, I get the message "Every request must have a bearer token in the Authorization header.".
But when I try to login first, I don't have token obviously.

Did I miss something?

**Comment by datamweb** [comment, marked as answer]
You must first send a request to **auth/token** as follows:

![Screenshot 2023-04-20 201249](https://user-images.githubusercontent.com/9530214/233432829-3393338d-1d57-4fb0-8066-5544087f1743.png)


Then you can access the addresses protected by **Tokens** Filter with the received code.

![Screenshot 2023-04-20 201418](https://user-images.githubusercontent.com/9530214/233433180-928a63ba-e427-486d-bce6-1dea0d75736e.png)
